src/preprocessing/scripts/filters.py

```diff
@@ -652,3 +656,32 @@ def strange_sub(example: dict[str, Any]) -> dict[str, Any]:
         return example
 
     return strange_sub
+
+
+# 文章中の省略記号の割合によるフィルタリング
+def has_valid_ending(max_ratio: float = 0.2) -> Callable[..., bool]:
+    ellipsis_pattern = re.compile(
+        r"(?:\.{2,}|…|､{2,}|、{2,}|－{2,}|ー{2,}|─{2,}|﹣{2,}|−{2,}|⋯|؞{2,}|⋮)",
```

Comment:
文末でマッチングするように修正をお願いします！
```suggestion
        r"(?:\.{2,}|…|､{2,}|、{2,}|－{2,}|ー{2,}|─{2,}|﹣{2,}|−{2,}|⋯|؞{2,}|⋮)$",
```